src/security/PeerOptions.cc

```diff
@@ -552,6 +552,7 @@ Security::PeerOptions::parseFlags()
         { SBuf("NO_DEFAULT_CA"), SSL_FLAG_NO_DEFAULT_CA },
         { SBuf("DELAYED_AUTH"), SSL_FLAG_DELAYED_AUTH },
         { SBuf("DONT_VERIFY_PEER"), SSL_FLAG_DONT_VERIFY_PEER },
+        { SBuf("CONDITIONAL_VERIFY"), SSL_FLAG_CONDITIONAL_VERIFY },
```

Comment:
For consistency with the DELAYED_AUTH flag this should probably be named OPTIONAL_AUTH.

Reply:
Alright. I wasn't sure about the name. "optional" like NGINX feels odd given it makes it sound like the verification is optional rather than being based on existence... but the SSL flag felt too verbose.

Reply:
I agree that `_AUTH` suffix is a good idea and that "optional" is a bit misleading. I can suggest `CONDITIONAL_AUTH` or even `IF_RECEIVED_CERTIFICATE_AUTH`. Pick what you think works best.